core/base/src/TApplication.cxx

```diff
@@ -1073,6 +1085,68 @@ void TApplication::Help(const char *line)
+      
+      if (strippedCommand == "edit") {
+         Printf("\nROOT terminal keyboard shortcuts (GNU-readline style).");
+         #ifdef R__MACOSX
+         #define FOOTNOTE " *"
+         Printf("* This command might be intercepted by macOS predefined system shortcuts.")
+         // https://apple.stackexchange.com/questions/18043/how-can-i-make-ctrlright-left-arrow-stop-changing-desktops-in-lion
+         #else
+         #define FOOTNOTE ""
+         #endif
+         Printf("==============================================================================");
+         Printf("   Alt+Backspace       : delete previous word" FOOTNOTE);
+         Printf("   Arrow Down          : navigate downwards in command history");
+         Printf("   Arrow Left          : move cursor left");
+         Printf("   Arrow Right         : move cursor right");
+         Printf("   Arrow Up            : navigate upwards in command history");
+         Printf("   Backspace           : remove previous character");
+         Printf("   CTRL+A              : move cursor to beginning of line");
+         Printf("   CTRL+Arrow Left     : jump to previous word" FOOTNOTE);
+         Printf("   CTRL+Arrow Right    : jump to next word" FOOTNOTE);
+         Printf("   CTRL+B              : move cursor left");
+         Printf("   CTRL+C              : send kSigInt interrupt signal");
+         Printf("   CTRL+D              : remove next character (or exit ROOT if empty line)");
+         Printf("   CTRL+Del            : remove next word" FOOTNOTE);
+         Printf("   CTRL+E              : move cursor to end of line");
+         Printf("   CTRL+F              : move cursor right");
+         Printf("   CTRL+G              : move cursor right");
+         Printf("   CTRL+H              : remove previous character");
+         Printf("   CTRL+I              : autocomplete command or print suggestions");
+         Printf("   CTRL+J              : execute command");
+         Printf("   CTRL+K              : remove all characters between cursor and end of line");
+         Printf("   CTRL+L              : clear screen");
+         Printf("   CTRL+M              : execute command");
+         Printf("   CTRL+N              : navigate downwards in command history");
+         Printf("   CTRL+O              : select command of the string-based history search (also Enter)");
+         Printf("   CTRL+P              : navigate upwards in command history");
+         Printf("   CTRL+Q              : ");
+         Printf("   CTRL+R              : reverse string-based history search");
+         Printf("   CTRL+T              : clear screen");
+         Printf("   CTRL+U              : remove all characters between cursor and start of line");
+         Printf("   CTRL+W              : delete previous word");
+         Printf("   CTRL+Y              : paste clipboard content (also CTRL+Shift+V)");
+         Printf("   CTRL+Z              : send kSigStop pause job signal");
```

Comment:
Is it not something general ? not ROOT specific ?

Reply:
in many text editors, CTRL+Z works as 'undo' shortcut, so even if it is also common as sigstop, I think it is still good to point it out.